### ISSUE TITLE
Docker image for version 3.19.0 contains crowdin-cli version 3.18.0

### ISSUE THREAD

**Issue by chriswhite-depop** [crowdin/crowdin-cli#745]
**Describe the bug**

The docker image for crowdin-cli version `3.19.0` contains crowdin-cli version `3.18.0`.

**To Reproduce**
1. Pull the `crowdin/cli:3.19.0` docker image
```
docker pull crowdin/cli:3.19.0
```
2. Check the crowdin CLI version
```
docker run DOCKER_IMAGE_ID crowdin --version
```

**Expected behavior**
`crowdin/cli:3.19.0` should contain crowdin-cli version `3.19.0`.

**Comment by andrii-bodnar** [comment]
Hi @chriswhite-depop, thanks for pointing this out!

There was an issue that happened during the deployment process. I just redeployed the `latest` tag of the Docker image which already contains the `3.19.0` version. Unfortunately, it's not possible to re-deploy the `3.19.0` tag.

Could you please use the `latest` tag until the next CLI release? Thanks and sorry for the inconvenience!

**Comment by andrii-bodnar** [comment]
Fixed in [3.19.1](https://github.com/crowdin/crowdin-cli/releases/tag/3.19.1)
